libupload/gradle.properties

```diff
@@ -0,0 +1,5 @@
+VERSION_NAME=0.1.0-SNAPSHOT
```

Comment:
i don't think we necessarily need to release upload library, i also think we could probably be just fine with implementing it as part of the internal package inside `libtelemetry`